client/state/help/selectors.js

```diff
@@ -1,25 +1,48 @@
 /** @format */
 
 /**
- * Internal dependencies
+ * External dependencies
  */
+import { get } from 'lodash';
 
+/**
+ * Internal dependencies
+ */
 import { getSelectedOrPrimarySiteId } from 'state/selectors';
 import { getSite } from 'state/sites/selectors';
 
 export const getHelpSiteId = state => state.help.selectedSiteId;
 
+/*
+ * Returns the site the customer wishes to request help for. Returns in order of preference:
+ *  - The site the customer has explicitly selected
+ *  - A selected site
+ *  - The primary site (which may not be available).
```

Comment:
It's probably worth explicitly documenting that this falls back to the first site if none of the other options are available.